lib/initializer.rb

```diff
@@ -0,0 +1,47 @@
+require_relative './game.rb'
+require_relative './board/board.rb'
+require_relative './board/board_checker.rb'
+require_relative './players/ai.rb'
+require_relative './players/human.rb'
+require_relative './players/player.rb'
+require_relative './ui/user_interface.rb'
+require_relative './ui/script.rb'
+
+def initializer(size, empty_char)
+  board = Board.new(size, empty_char)
+  checker = BoardChecker.new(size, empty_char)
```

Comment:
[Changed this to take the board](https://github.com/saralein/tic-tac-toe_ruby/pull/7/files#diff-6bfad8daa181b5e0a43f92245aa59e75R14)